src/components/button/button.sandbox.html

```diff
@@ -15,9 +15,10 @@ <h2 class="m-u--no-margin">L'utilsation de v-html dans un bouton et d'un &lt;m-i
             </m-button>
 
             <m-button @click="toogleAfficher()"
-                      v-if="!afficherbouton1">
+                      v-if="!afficherbouton1"
+                      key="username-input">
                 patate2
-            </m-button key="username-input">
```

Comment:
Hein? Ça devait rien faire un key sur un tag html qui ferme. Je ne verrai pas pourquoi il faudrait le remettre sur m-button. Un key a rapport à un v-for en plus....

Reply:
LOL c'est drôle ça

Reply:
@vidal7 oui c'est hors du scope du PR mais ce sandbox montre un bug bizzard qu'il y a eu dans admission, le seul moyen de le fixer est avec le :key 

https://fr.vuejs.org/v2/guide/conditional.html#Controle-des-elements-reutilisables-avec-key